src/libraries/System.Private.CoreLib/src/System/Numerics/IBinaryNumber.cs

```diff
@@ -10,6 +10,9 @@ public interface IBinaryNumber<TSelf>
           INumber<TSelf>
         where TSelf : IBinaryNumber<TSelf>
     {
+        /// <summary>Gets an instance of the binary type in which all bits are set.</summary>
+        static abstract TSelf AllBitsSet { get; } // TODO: add the DIM once https://github.com/dotnet/linker/issues/2865 is fixed
```

Comment:
Do we have a runtime issue tracking all of the APIs that need to be made DIMs once this bug is fixed?  Presumably this is a must-do for .NET 7?

Reply:
The other APIs are already DIM where relevant. This is a new scenario specific to properties.

Reply:
> The other APIs are already DIM where relevant.

Oh.  We're not making APIs like IParseable.Parse DIMs?
https://github.com/dotnet/runtime/blob/3fb9511bf78a5de5cb73bad8fd50a69c691d61cd/src/libraries/System.Private.CoreLib/src/System/IParsable.cs#L20

Reply:
(or agree that we just always throw `FormatException` and expect users that need to differentiate to always override).

Reply:
The fact that I'm the first person to run into this implies this is the first static virtual property somebody has tried to use. This is surprising and suspicious. I wonder if I'm just the first one to realize the DIM isn't being overridden by inheriting classes. Regardless, yes, I think fixing the bug should have relatively high priority.